Resources/templates/CommonAdmin/object_actions.php.twig

```diff
@@ -35,17 +35,23 @@
 
     {% spaceless %}
         {{ echo_spaceless() }}
+        <div {% if builder.yamlKey is same as('list') or builder.yamlKey is same as('nested_list') %}
```

Comment:
addition div is wrong, span would be better, but i will remove addional tag at all, which is the best

about dynamic forms:
I think, the correct behavior would be:
1) confirmation is needed -> show bs modal and POST a form with or without csrf protection if confirmed
2) no confirmation and csrf protection -> POST a dynamic form with csrf
3) no confirmation and no protection -> do not change, href link is followed

Do you agree?